app/models/publication-flow.js

```diff
@@ -46,57 +45,4 @@ export default class PublicationFlow extends Model {
     }
     return `${this.publicationNumber}`;
   }
-
-  get translationRequestsTotal() {
-    return this.publicationService.getPublicationCounts(this.id).then((totals) => {
-      const openVertalingen = this.publicationService.getPublicationCountsPerTypePerStatus(totals, CONFIG.ACTIVITY_TYPES.vertalen.url, CONFIG.ACTIVITY_STATUSSES.open.url);
-      const closedVertalingen = this.publicationService.getPublicationCountsPerTypePerStatus(totals, CONFIG.ACTIVITY_TYPES.vertalen.url, CONFIG.ACTIVITY_STATUSSES.closed.url);
```

Comment:
De method in service kunnen we ook mee opkuisen (indien geen gebruik meer op andere plaatsen)